src/dfi/mn_checks.cpp

```diff
@@ -1074,6 +1076,15 @@ Res CPoolSwap::ExecuteSwap(CCustomCSView &view,
                 if (!res) {
                     return res;
                 }
+
+                if (lastSwap) {
+                    LogPrint(BCLog::SWAPRESULT,
```

Comment:
Let's do a pre-check with IsLoggingEnabled, so that ScriptToString and any numerics to string due to ToString is avoided unnecessarily.